src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -46,28 +44,37 @@ trait ThurloeService extends HttpService with LazyLogging {
   }
 
   val queryRoute = path(ThurloePrefix) {
-    parameterSeq { parameters =>
-      get {
+    get {
+      parameterSeq { parameters =>
         val thurloeQuerySpec = ThurloeQuery(parameters)
-        thurloeQuerySpec.unrecognizedFilters match {
-          case Some(invalidFilters) =>
-            respondWithStatus(StatusCodes.BadRequest) {
-              complete {
-                "Bad query parameter(s): " + invalidFilters.mkString(",")
-              }
+
+        if (thurloeQuerySpec.isEmpty) {
+          respondWithStatus(StatusCodes.BadRequest) {
+            complete {
+              "No query parameters specified"
```

Comment:
ideally, this would return a parsable json object instead of just a string, but I wouldn't hold the PR just for that

Reply:
+1  I was just keeping with the style of the other failure endpoints in thurloe